### DISCUSSION TITLE
3주차 회고

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#77] [회고]
KPT 회고는 세가지 관점으로 회고를 진행합니다.

KEEP(현재 만족하고 있고, 계속 이어가길 바라는 점)
PROBLEM(불편을 느껴 개선이 필요한 점)
TRY(PROBLEM의 해결책으로 다음 회고까지 실행할 점)

아래 양식을 복사해 댓글로 작성해주시면 됩니다.
```
### KEEP

### PROBLEM

### TRY
```

**Comment by minyoung0503** [comment]
### KEEP
소통이 빠르게 잘 된다.
작업속도가 빠르다.
작은 일이라도 다같이 의논하고 결정한다.
팀 분위기가 밝고 적극적이다.

### PROBLEM
코드리뷰를 많이 하지 못했다.
작업속도가 빨라서 조급해진다.

### TRY
코드리뷰 적극적으로 하기
더 열심히 하기

**Reply by Seoin02** [reply]
코드리뷰에 부담 안 가지셔도 됩니다~ 할 수 있는 선에서 열심히 해봅시당ㅎㅎ

**Comment by Seoin02** [comment]
### KEEP
- 팀원들이 어려운 문제를 함께 해결하는 게 좋았다. 다들 감사해요!
- 회의에서 팀원 모두 의견이나 질문을 적극적으로 말한다. 이로 인해 문제가 생길 수 있는 부분이나 애매한 지점을 확실히 정하고 작업을 진행하고 있다.
- 데이터 통신과 관련해 FE와 BE의 다른 입장도 잘 조율하고 있다.
- 디자이너, BE가 빠른 속도로 작업하고 있다. 감사합니다!
- 깃허브 Discussions를 활용해 데일리스크럼을 잘 공유하고 있다.
- 다들 깃 컨벤션, 코드 컨벤션을 잘 지키고 있다.
- 팀원들이 PR을 잘 확인한다. 서로 힘이 되는 말을 주고 받는 게 좋다.

### PROBLEM
- 생각보다 페이지 작업이 오래 걸린다.
- 하나의 PR에 많은 코드를 올리고 있다.

### TRY
- 페이지 전체를 구현하고 PR을 올리기 보다 이슈와 PR을 작은 단위로 쪼개야겠다.

**Comment by devwqc** [comment]
### KEEP
- 자신이 맡은 부분이 아니어도 같이 고민해 주고 신경 써준다.
- 어떤 이슈가 생겼을 때 서로를 탓하지 않고 같이 해결하려고 노력한다.
- 카카오톡을 통한 빠른 소통이 된다.

### PROBLEM
- 우리 팀은 야근 금지, 일요일 휴식이 기본인데 잘 지켜지지 않는 거 같다.

### TRY
- 휴식을 잘 취할 것.

**Comment by q45402sk** [comment]
### KEEP
팀원 모두가 팀미팅 시간을 잘 지킨다. 
열심히 한다. 

### PROBLEM
PR 확인을 많이 못했다.

### TRY
PR 확인을 바로바로 하려고 노력한다.
